app/controllers/hackathons/submissions_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Hackathons::SubmissionsController < ApplicationController
+  def index
+    @hackathons = Hackathon.not_approved.where applicant: Current.user
```

Comment:
Notes from review on zoom:
- Not required, but it could be nice to also see the approved applications